client/common/file.c

```diff
@@ -191,6 +191,14 @@ struct rdp_file
 
 	LPSTR ResourceProvider; /* resourceprovider */
 
+	LPSTR wvd;                  /* wvd endpoint pool */
```

Comment:
This could have a better name (what is the **w**indows **v**irtual **d**estkop string for?)

Reply:
ok, renamed to `WvdEndpointPool`